com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -237,7 +237,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed support of Distortion with MSAA
 - Fixed contact shadow debug views not displaying correctly upon resizing of view.
 - Fixed an error when deleting the 3D Texture mask of a local volumetric fog volume (case 1339330).
-- Fixed some aliasing ussues with the volumetric clouds.
```

Comment:
Looks like the minus sign got deleted here somehow? I don't want to break HD's changelog. And while you're at it, you could spell "issues" correctly. =)

Reply:
Done!